### DISCUSSION TITLE
Have MOOSE output files outside of moose project folder

### DISCUSSION THREAD

**Discussion by jackeggemeyer** [idaholab/moose#27563] [Q&A General] [answered]
I am trying to output outside of my project folder named Tarpon. I am trying to output to a folder I use for some MATLAB post-processing; however, moose will not identify the home path. Instead of going to:
 Home/Main/Data

MOOSE creates a new file path inside itself:
Home/projects/tarpon/Home/Main/Data

How can I make moose output outside of itself? There is not much documentation on file_base

[Outputs]
  file_base = Home/Main/Data/3d2
  [./exodus]
    type = Exodus
    interval = 1
    #sync_times = '1000 1200'
    file_base = Home/Main/Data/3d2
  [../]
  csv = true
  checkpoint = true
[]

**Comment by GiudGiud** [comment, marked as answer]
Hello

You can either use `../../` etc to get out of the folder where the input is run.
Or you can try an absolute path `~/Home/Main/Data`

Either should be supported though I have only tested the former

Guillaume

**Reply by jackeggemeyer** [reply]
Thank you. Using "../../" worked perfectly. 

I tried using the absolute path first, but it kept just making a folder named "~".
